test/unit/examples/contextRequests/updateContextTimestampTimezone.json

```diff
@@ -0,0 +1,41 @@
+{
+  "contextElements": [
+    {
+      "type": "Light",
+      "isPattern": "false",
+      "id": "light1",
+      "attributes": [
+        {
+          "name": "state",
+          "type": "Boolean",
+          "value": "true",
+          "metadatas": [
+            {
+              "name": "TimeInstant",
+              "type": "ISO8601",
+              "value": "2015-08-05T00:35:01.468Z"
```

Comment:
As far as I understand, the functionality is about supporting timezones. However, in the JSON included in this PR "Z" is always used in mocked requests to CB.

Does it means that the IOTAgent is translation timezone and will always send datetimes in UTC to CB?

Reply:
Sorry, to be honest, I did not know that the Z corresponds to a timezone. In fact, I thought that it was just part of the ISO 8601.
I have changed this and now formats are correctly formatted according to the standards and reflecting the timezone.

Reply:
Fixed with 0f341d164437b4f2b9876ba8acf56d25afa5cefa

Reply:
I'm not an expert in ISO8601, but I think that Z equals to UTC (a kind of "virtual timezone" where time is alwyas in UTC, no matter summer/winter time changes, etc.)

Reply:
You're right, in fact, it means `Zulu time`: https://www.timeanddate.com/time/zones/z

:)